tests/Handler/DownloadedRecoveryCodeHandlerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Connect Holland Secure JWT package and distributed under the terms of the MIT License.
+ * Copyright (c) 2020 Connect Holland.
+ */
+
+namespace ConnectHolland\SecureJWTBundle\Tests\Handler;
+
+use ConnectHolland\SecureJWTBundle\Entity\RecoveryCode as RecoveryCodeEntity;
+use ConnectHolland\SecureJWTBundle\Handler\DownloadedRecoveryCodeHandler;
+use ConnectHolland\SecureJWTBundle\Message\DownloadedRecoveryCode;
+use ConnectHolland\SecureJWTBundle\Tests\Fixture\User;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Lexik\Bundle\JWTAuthenticationBundle\Security\Authentication\Token\JWTUserToken;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage;
+use Symfony\Component\Security\Core\Exception\RuntimeException;
+
+class DownloadedRecoveryCodeHandlerTest extends TestCase
+{
+    public function testSetCodesDownloadedSuccessful(): void
```

Comment:
You are forgetting to test the important element of this method: are the recovery codes set to downloaded?